static/js/publisher/release/releasesController.js

```diff
@@ -555,5 +560,33 @@ ReleasesController.propTypes = {
   snapName: PropTypes.string.isRequired,
   channelMapsList: PropTypes.array.isRequired,
   releasesData: PropTypes.object.isRequired,
-  options: PropTypes.object.isRequired
+  options: PropTypes.object.isRequired,
+
+  revisions: PropTypes.object,
+  state: PropTypes.object,
+  updateRevisions: PropTypes.func
+};
+
+const mapStateToProps = state => {
+  return {
+    revisions: state.revisions
+  };
+};
+
+const mapDispatchToProps = dispatch => {
+  return {
+    // TODO: extract to outside actions module
+    updateRevisions: revisions => {
+      dispatch({
```

Comment:
Might be worth considering using action creators to decouple action payload from the component.

Reply:
Sure, slowly getting there!